go.sum

```diff
@@ -221,8 +221,12 @@ github.com/datawire/dlib v1.3.0 h1:KkmyXU1kwm3oPBk1ypR70YbcOlEXWzEbx5RE0iRXTGk=
 github.com/datawire/dlib v1.3.0/go.mod h1:NiGDmetmbkBvtznpWSx6C0vA0s0LK9aHna3LJDqjruk=
 github.com/datawire/dtest v0.0.0-20210928162311-722b199c4c2f h1:Yz+xVYWkSARvfJ0e3LWwF1N5F+zJI9pCZ9R41OtkrKE=
 github.com/datawire/dtest v0.0.0-20210928162311-722b199c4c2f/go.mod h1:D/jb71PHlfUjfNSUVpXQAscXsXy3x+CY+c52lPnW2qU=
-github.com/datawire/go-mkopensource v0.0.0-20220218163159-cc298d9fabc4 h1:brHMnjdzg0cFM8HGwXyYhpEK1esLHPmrFu1N/cRFRPA=
-github.com/datawire/go-mkopensource v0.0.0-20220218163159-cc298d9fabc4/go.mod h1:u8J6vyaKZZSKl/sFtiQLV8I6n8xqDdPOXdY9mAQzmNg=
+github.com/datawire/go-mkopensource v0.0.2 h1:r7b/jtIQPF0pvwrE+LiIiLBg6upuNsKRSxXSkygviiA=
+github.com/datawire/go-mkopensource v0.0.2/go.mod h1:6WIetr8QL8XpzFUjJAb6kA/yQALBOGxUCd10UXEQwyY=
+github.com/datawire/go-mkopensource v0.0.3 h1:YZUG3HXMRewrULt8c+it9dbJSTj5jFfeKZ6AsX0SQqk=
+github.com/datawire/go-mkopensource v0.0.3/go.mod h1:6WIetr8QL8XpzFUjJAb6kA/yQALBOGxUCd10UXEQwyY=
+github.com/datawire/go-mkopensource v0.0.4 h1:l7FHSMjY4PjI/GTIHuf7otS3jYkBFVtj6+oh7dKJsX8=
+github.com/datawire/go-mkopensource v0.0.4/go.mod h1:6WIetr8QL8XpzFUjJAb6kA/yQALBOGxUCd10UXEQwyY=
```

Comment:
Why does this include so many versions?

Reply:
I've removed old references.

Reply:
The `go.sum` shouldn't be manually manipulated and it is to ensure that what runs locally also runs the same in CI. It is maintained by `go mod tidy` or in the case of emissary-ingress we use `make go-mod-tidy` because it takes care of running it recursively in multiple directories.

Here is an explanation on the reason for multiple entries in the file: https://stackoverflow.com/questions/69825533/why-does-go-sum-include-so-many-older-packages